docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
+moisture diffusion flux (only included is ``evaporation`` is true). The ocean_flux is an optional term and is zero if the Q-flux is not used.
+
+If ``do_calc_eff_heat_cap`` is True, the land_sea_heat_capacity is used at each timestep to compute the surface temperature using the following equation:
+
+land_sea_heat_capacity * dTs/dt = - corrected_flux - t_surf_dependence * dTs
+
+The change in surface temperature in this timestep (dTs/dt) times the heat capacity of the surface is equal to the net flux into the surface at the given timestep (- corrected_flux) plus the change in the net flux 
+into the surface caused by the change in surface temperature (- t_surf_dependence * dTs).
+
+This is simplified by defining eff_heat_capacity as land_sea_heat_capacity + t_surf_dependence * dt, and then Ts is updated using
+
+eff_heat_capacity * dTs/dt = - corrected_flux.
+
+Additional Q-flux
+^^^^^^^^^^^^^^^^^^^^
+The slab ocean model only communicates between grid-boxes in the vertical (i.e. air-sea exchange) but does not represent any horizontal transport (i.e. no north-south or east-west communications between grid cells). 
+An idealised horizontal transport can be included using an ocean heat flux (q-flux). Atmospheric heat transport is more realistic with an ocean heat transport.
+
+To create a qflux file, run a prescribed run (i.e. a control) using either observations, AMIP or similar. The prescribed SST field and the surface fluxes (from this control run)
+are then used to compute the Q-flux by running the offline python script: src/extra/python/scripts/calculate_qflux/calculate_qflux.py. This script produces a .nc file which is then passed into the model 
+(qflux_file_name).
+
+See Q-flux options below for namelist options. Note that the q-flux is only relevant for slab ocean experiments (not fixed or prescribed SST runs). Also note that if the MiMA radiation code is used then the 
+Q-flux is implemented following Merlis et al 2013 [MerlisEtAl2013]_
+
+More information on the method for Q-flux can be found in Russel et al 1985 [RusselEtAl1985]_
+
+
+Namelist options
+----------------
+
++-------------------+------------------------------------------------------------+---------+
+| Option            | Summary                                                    |Default  |
++===================+============================================================+=========+
+|``evaporation``    |Switch for surface evaporation.                             |``True`` |
++-------------------+------------------------------------------------------------+---------+
+|``depth``          |Mixed layer depth.                                          | ``40.0``|
++-------------------+------------------------------------------------------------+---------+
+
+Q-flux options
+^^^^^^^^^^^^^^^^^^^^
+If ``do_qflux`` is True, use ``qflux_amp`` and ``qflux_width`` to calculate a time-independent surface q-flux.
```

Comment:
Could you add a sentence above in the q-flux description to include how the qflux_amp and qflux_width work?

Reply:
Okay I get why there are two tables now. Could you separate the text above into two sections corresponding to each table please? Might make it more obvious.